packages/database/src/test/setup.ts

```diff
@@ -1 +1,9 @@
+import { getLegacyTestDatabase } from '@l2beat/database-legacy'
+
 process.env.NODE_ENV = 'test'
+
```

Comment:
We didn't include
```js
waitForExpect.defaults.timeout = 1500
waitForExpect.defaults.interval = 10
```
there, was it on purpose?

Reply:
We don't really use it outside of two specific tests in the backend package